packages/java/maven-plugin/src/main/java/dev/hilla/maven/EngineGenerateMojo.java

```diff
@@ -44,7 +44,8 @@ public void execute() throws EngineGenerateMojoException {
             for (var classPathItem : classPath) {
                 urls.add(new File(classPathItem).toURI().toURL());
             }
```

Comment:
I guess you could use `stream()` here.

Reply:
The `toURL()` method throws `MalformedURLException`, and exception handling for Stream items is not nice. Luckily, with good'ol plain code this exception is handled with the surrounding `try / catch`.

Reply:
ah, I see. Meh, that broken Stream...